Sming/SmingCore/DateTime/DateTime.h

```diff
@@ -112,14 +112,26 @@ class DateTime
 	 *  @note   Also supports obsolete RFC 850 date format, e.g. Sunday, 06-Nov-94 08:49:37 GMT where 2 digit year represents range 1970-2069
 	 *  @note   GMT suffix is optional and is always assumed / ignored
 	 */
-	bool parseHttpDate(const String& httpDate);
+	bool fromHttpDate(const String& httpDate);
+
+	/** @brief  Parse a HTTP full date and set time and date
+	 *  @param  httpDate HTTP full date in RFC 1123 format, e.g. Sun, 06 Nov 1994 08:49:37 GMT
+	 *  @retval bool True on success
+	 *  @note   Also supports obsolete RFC 850 date format, e.g. Sunday, 06-Nov-94 08:49:37 GMT where 2 digit year represents range 1970-2069
+	 *  @note   GMT suffix is optional and is always assumed / ignored
+	 *  @deprecated Use 'fromHttpDat' instead
```

Comment:
`fromHttpDat` change to `fromHttpDate`.

Reply:
Fixed in working copy. Will push when coding style checked and committed.

Reply:
Fixed in last push.